test-data/unit/check-basic.test

```diff
@@ -232,6 +232,13 @@ a = __file__ # type: A  # E: Incompatible types in assignment (expression has ty
 class A: pass
 [builtins fixtures/primitives.pyi]
 
+[case testModule__path__]
+import typing
+x = __path__ # type: list
+a = __path__ # type: A  # E: Incompatible types in assignment (expression has type "list", variable has type "A")
```

Comment:
How about just using a `reveal_type` on `__path__`?

Reply:
I agree, I was only going with the flow of other similar tests. Should them also use `reveal_type` on `__name__` and etc?

Reply:
As `builtins.list` is partially typed, `reveal_type` reveals `builtins.list[Any]` instead of `builtins.list`.